pkg/plugins/config/config.go

```diff
@@ -53,5 +55,6 @@ func NewPluginManagementCfg(devMode bool, pluginsPath string, pluginSettings set
 		AngularSupportEnabled:  angularSupportEnabled,
 		HideAngularDeprecation: hideAngularDeprecation,
 		ForwardHostEnvVars:     forwardHostEnvVars,
+		GcomToken:              gcomToken,
```

Comment:
this `PluginManagementCfg` is meant to be the config to manage the lifecycle of plugins or to forward config to the plugins. As far as I can tell, the GCOM token is not needed for that? Also, I am not seeing anything using this.

Reply:
Thank for pointing this out. You're right, I forgot to add the part where the token is used. This is necessary to setup the gcom token in pkg/plugins/repo/service.go